src/sft-minter.ts

```diff
@@ -371,6 +426,10 @@ export class SftMinter extends Minter {
       data.addArg(new U64Value(lockPeriod));
     }
 
+    if (donationPercentage) {
```

Comment:
If donationPercentage is not given as an argument it should be still put as an arg in the tx (as 0) as it is an obligatory argument, right?